.github/mergify.yml

```diff
@@ -15,6 +15,8 @@ pull_request_rules:
       - check-success=test(ruby-3.0, rails_60)
       - check-success=test(ruby-3.0, rails_61)
       - check-success=test(ruby-3.0, rails_70)
+      - check-success=test(ruby-3.1, rails_61)
```

Comment:
```suggestion
      - check-success=test(ruby-3.1, rails_60)
      - check-success=test(ruby-3.1, rails_61)
```

Reply:
Good catch! I've missed that one. Added :)

Reply:
Awesome thanks!